Specs/Core/loadKTX2Spec.js

```diff
@@ -0,0 +1,295 @@
+import { loadKTX2 } from "../../Source/Cesium.js";
+import { KTX2Transcoder } from "../../Source/Cesium.js";
+import { PixelFormat } from "../../Source/Cesium.js";
+import { Resource } from "../../Source/Cesium.js";
+import { RuntimeError } from "../../Source/Cesium.js";
+
+describe("Core/loadKTX2", function () {
+  it("throws with no url", function () {
+    expect(function () {
+      loadKTX2();
+    }).toThrowDeveloperError();
+  });
+
+  it("throws with unknown supported formats", function () {
+    expect(function () {
+      KTX2Transcoder.transcode(new Uint8Array());
+    }).toThrowDeveloperError();
+  });
+
+  it("throws if loadKTX2 is with invalid url", function () {
+    var testUrl = "http://example.invalid/testuri";
+    var promise = loadKTX2(testUrl);
+    return promise
+      .then(function (value) {
+        fail();
+      })
+      .otherwise(function (error) {
+        expect(error).toBeDefined();
```

Comment:
@lilleyse This throws a 404 in Sandcastle but an empty error object in spec... I thought it might be better to add `expect(error.statusCode).toBe(404)` or something similar but wasn't sure if this was possible.

Reply:
Hm yeah I'm seeing that too. It doesn't need to hold up the PR.